projects/plugins/jetpack/extensions/plugins/publicize/components/connection/index.js

```diff
@@ -17,8 +17,8 @@ import { includes } from 'lodash';
 /**
  * Internal dependencies
  */
-import getSiteFragment from '../../shared/get-site-fragment';
-import { SocialServiceIcon } from '../../shared/icons';
+import getSiteFragment from '../../../../shared/get-site-fragment';
+import { SocialServiceIcon } from '../../../../shared/icons';
```

Comment:
well... I'm not sure. Maybe the icons should be a package by itself, in a similar way that exists the `@wordpress/icons` package.

```es6
import { getSiteFragment } from '@jetpack/element'; // <- or something like that
import { SocialServiceIcon } from '@jetpack/icons';
```

Reply:
I think that would make sense, especially since we now have more and more packages, all set up for distribution:
https://github.com/Automattic/jetpack/tree/master/projects/js-packages